$Id: 099k_Compose_fix_latin1_UTF8.diff 471 2005-08-02 03:47:27Z dnusinow $

This patch by Denis Barbier.

WARNING: do not recode this file, it contains UTF-8 characters.

The X11 protocol states that Unicode keysyms are in the range
0x01000100 - 0x0110FFFF.  If the result of composing characters
is a Unicode codepoint, X returns the corresponding Unicode
keysym, which is its Unicode codepoint augmented by 0x01000000.
Latin-1 characters must not appear with their Unicode codepoints
in compose files, otherwise the returned composed character lies
in the range 0x01000000 - 0x010000FF which is not valid.

There are two solutions: either fix composing routines to return
0xZZ instead of 0x010000ZZ (where Z is an hexadecimal digit),
or replace U00ZZ by their corresponding keysyms in compose files.
The latter is more logical and less error prone, so compose
files will be patched.

Many applications accept these invalid Unicode keysyms, but few of
them don't, most notably xemacs.  Only UTF-8 locales are affected.

This has been fixed very recently in XFree86 CVS (but not xorg),
but for licensing reasons, this patch is not grabbed.
Instead automatic conversion is performed by:
  sed -e '/XK_LATIN1/,/XK_LATIN1/!d' /usr/X11R6/include/X11/keysymdef.h \
  | grep -v deprecated | grep 0x0 \
  | sed -e 's/0x0/U0/' -e 's/XK_//' \
  | awk '{ printf "s/\\b%s\\b/%s/ig\n", $3, $2; }' > sedfile
  for f in *.UTF-8
  do
    sed -f sedfile $f > $f.tmp && mv $f.tmp $f
  done

Update: pt_BR.UTF-8 part removed. We replace it by a simple #include with a 
couple of additional entries later anyway. (Eugene Konev)

Index: libx11/nls/en_US.UTF-8/Compose.pre
===================================================================
--- libx11.orig/nls/en_US.UTF-8/Compose.pre
+++ libx11/nls/en_US.UTF-8/Compose.pre
@@ -92,7 +92,7 @@
 
 <Multi_key> <period> <greater>   	: "›"   U203a # SINGLE RIGHT-POINTING ANGLE QUOTATION MARK
 <Multi_key> <period> <less>      	: "‹"   U2039 # SINGLE LEFT-POINTING ANGLE QUOTATION MARK
-<Multi_key> <period> <period>    	: "·"   U00B7 # MIDDLE DOT
+<Multi_key> <period> <period>    	: "·"   periodcentered # MIDDLE DOT
 <Multi_key> <exclam> <asciicircum> 	: "¦"   brokenbar # BROKEN BAR
 <Multi_key> <exclam> <exclam>    	: "¡"   exclamdown # INVERTED EXCLAMATION MARK
 <Multi_key> <p> <exclam>         	: "¶"   paragraph # PILCROW SIGN
@@ -156,12 +156,12 @@
 <Multi_key> <E> <equal>          	: "€"   EuroSign # EURO SIGN
 <Multi_key> <equal> <E>          	: "€"   EuroSign # EURO SIGN
 
-<Multi_key> <bar> <c>            	: "¢"   U00A2 # CENT SIGN
-<Multi_key> <c> <bar>            	: "¢"   U00A2 # CENT SIGN
-<Multi_key> <c> <slash>          	: "¢"   U00A2 # CENT SIGN
-<Multi_key> <slash> <c>          	: "¢"   U00A2 # CENT SIGN
-<Multi_key> <L> <minus>          	: "£"   U00a3 # POUND SIGN
-<Multi_key> <minus> <L>          	: "£"   U00a3 # POUND SIGN
+<Multi_key> <bar> <c>            	: "¢"   cent # CENT SIGN
+<Multi_key> <c> <bar>            	: "¢"   cent # CENT SIGN
+<Multi_key> <c> <slash>          	: "¢"   cent # CENT SIGN
+<Multi_key> <slash> <c>          	: "¢"   cent # CENT SIGN
+<Multi_key> <L> <minus>          	: "£"   sterling # POUND SIGN
+<Multi_key> <minus> <L>          	: "£"   sterling # POUND SIGN
 <Multi_key> <Y> <equal>          	: "¥"   yen # YEN SIGN
 <Multi_key> <equal> <Y>          	: "¥"   yen # YEN SIGN
 
@@ -429,191 +429,191 @@
 <Multi_key> <U100011B7> <U100011BB> 	: "ᇞ"   U11de # HANGUL JONGSEONG MIEUM-SSANGSIOS
 <Multi_key> <U100011BC> <U100011A9> 	: "ᇭ"   U11ed # HANGUL JONGSEONG IEUNG-SSANGKIYEOK
 
-<Multi_key> <comma> <minus>      	: "¬"   U00AC # NOT SIGN
-<Multi_key> <minus> <comma>      	: "¬"   U00AC # NOT SIGN
-<dead_circumflex> <Multi_key> <underscore> <a> 	: "ª"   U00AA # FEMININE ORDINAL INDICATOR
-<Multi_key> <asciicircum> <underscore> <a> 	: "ª"   U00AA # FEMININE ORDINAL INDICATOR
-<dead_circumflex> <Multi_key> <underbar> <a> 	: "ª"   U00AA # FEMININE ORDINAL INDICATOR
-<Multi_key> <asciicircum> <underbar> <a> 	: "ª"   U00AA # FEMININE ORDINAL INDICATOR
-<dead_circumflex> <2>            	: "²"   U00B2 # SUPERSCRIPT TWO
-<Multi_key> <asciicircum> <2>    	: "²"   U00B2 # SUPERSCRIPT TWO
-<dead_circumflex> <KP_Space>     	: "²"   U00B2 # SUPERSCRIPT TWO
-<Multi_key> <asciicircum> <KP_Space> 	: "²"   U00B2 # SUPERSCRIPT TWO
-<dead_circumflex> <KP_2>         	: "²"   U00B2 # SUPERSCRIPT TWO
-<Multi_key> <asciicircum> <KP_2> 	: "²"   U00B2 # SUPERSCRIPT TWO
-<dead_circumflex> <3>            	: "³"   U00B3 # SUPERSCRIPT THREE
-<Multi_key> <asciicircum> <3>    	: "³"   U00B3 # SUPERSCRIPT THREE
-<dead_circumflex> <KP_3>         	: "³"   U00B3 # SUPERSCRIPT THREE
-<Multi_key> <asciicircum> <KP_3> 	: "³"   U00B3 # SUPERSCRIPT THREE
-<Multi_key> <m> <u>              	: "µ"   U00B5 # MICRO SIGN
-<dead_circumflex> <1>            	: "¹"   U00B9 # SUPERSCRIPT ONE
-<Multi_key> <asciicircum> <1>    	: "¹"   U00B9 # SUPERSCRIPT ONE
-<dead_circumflex> <KP_1>         	: "¹"   U00B9 # SUPERSCRIPT ONE
-<Multi_key> <asciicircum> <KP_1> 	: "¹"   U00B9 # SUPERSCRIPT ONE
-<dead_circumflex> <Multi_key> <underscore> <o> 	: "º"   U00BA # MASCULINE ORDINAL INDICATOR
-<Multi_key> <asciicircum> <underscore> <o> 	: "º"   U00BA # MASCULINE ORDINAL INDICATOR
-<dead_circumflex> <Multi_key> <underbar> <o> 	: "º"   U00BA # MASCULINE ORDINAL INDICATOR
-<Multi_key> <asciicircum> <underbar> <o> 	: "º"   U00BA # MASCULINE ORDINAL INDICATOR
-<Multi_key> <1> <4>              	: "¼"   U00BC # VULGAR FRACTION ONE QUARTER
-<Multi_key> <1> <2>              	: "½"   U00BD # VULGAR FRACTION ONE HALF
-<Multi_key> <3> <4>              	: "¾"   U00BE # VULGAR FRACTION THREE QUARTERS
-<dead_grave> <A>                 	: "À"   U00C0 # LATIN CAPITAL LETTER A WITH GRAVE
-<Multi_key> <grave> <A>          	: "À"   U00C0 # LATIN CAPITAL LETTER A WITH GRAVE
-<combining_grave> <A>            	: "À"   U00C0 # LATIN CAPITAL LETTER A WITH GRAVE
-<dead_acute> <A>                 	: "Á"   U00C1 # LATIN CAPITAL LETTER A WITH ACUTE
-<Multi_key> <acute> <A>          	: "Á"   U00C1 # LATIN CAPITAL LETTER A WITH ACUTE
-<Multi_key> <apostrophe> <A>     	: "Á"   U00C1 # LATIN CAPITAL LETTER A WITH ACUTE
-<combining_acute> <A>            	: "Á"   U00C1 # LATIN CAPITAL LETTER A WITH ACUTE
-<dead_circumflex> <A>            	: "Â"   U00C2 # LATIN CAPITAL LETTER A WITH CIRCUMFLEX
-<Multi_key> <asciicircum> <A>    	: "Â"   U00C2 # LATIN CAPITAL LETTER A WITH CIRCUMFLEX
-<dead_tilde> <A>                 	: "Ã"   U00C3 # LATIN CAPITAL LETTER A WITH TILDE
-<Multi_key> <asciitilde> <A>     	: "Ã"   U00C3 # LATIN CAPITAL LETTER A WITH TILDE
-<combining_tilde> <A>            	: "Ã"   U00C3 # LATIN CAPITAL LETTER A WITH TILDE
-<dead_diaeresis> <A>             	: "Ä"   U00C4 # LATIN CAPITAL LETTER A WITH DIAERESIS
-<Multi_key> <quotedbl> <A>       	: "Ä"   U00C4 # LATIN CAPITAL LETTER A WITH DIAERESIS
-<dead_abovering> <A>             	: "Å"   U00C5 # LATIN CAPITAL LETTER A WITH RING ABOVE
-<Multi_key> <o> <A>              	: "Å"   U00C5 # LATIN CAPITAL LETTER A WITH RING ABOVE
-<dead_cedilla> <C>               	: "Ç"   U00C7 # LATIN CAPITAL LETTER C WITH CEDILLA
-<Multi_key> <comma> <C>          	: "Ç"   U00C7 # LATIN CAPITAL LETTER C WITH CEDILLA
-<dead_grave> <E>                 	: "È"   U00C8 # LATIN CAPITAL LETTER E WITH GRAVE
-<Multi_key> <grave> <E>          	: "È"   U00C8 # LATIN CAPITAL LETTER E WITH GRAVE
-<combining_grave> <E>            	: "È"   U00C8 # LATIN CAPITAL LETTER E WITH GRAVE
-<dead_acute> <E>                 	: "É"   U00C9 # LATIN CAPITAL LETTER E WITH ACUTE
-<Multi_key> <acute> <E>          	: "É"   U00C9 # LATIN CAPITAL LETTER E WITH ACUTE
-<Multi_key> <apostrophe> <E>     	: "É"   U00C9 # LATIN CAPITAL LETTER E WITH ACUTE
-<combining_acute> <E>            	: "É"   U00C9 # LATIN CAPITAL LETTER E WITH ACUTE
-<dead_circumflex> <E>            	: "Ê"   U00CA # LATIN CAPITAL LETTER E WITH CIRCUMFLEX
-<Multi_key> <asciicircum> <E>    	: "Ê"   U00CA # LATIN CAPITAL LETTER E WITH CIRCUMFLEX
-<dead_diaeresis> <E>             	: "Ë"   U00CB # LATIN CAPITAL LETTER E WITH DIAERESIS
-<Multi_key> <quotedbl> <E>       	: "Ë"   U00CB # LATIN CAPITAL LETTER E WITH DIAERESIS
-<dead_grave> <I>                 	: "Ì"   U00CC # LATIN CAPITAL LETTER I WITH GRAVE
-<Multi_key> <grave> <I>          	: "Ì"   U00CC # LATIN CAPITAL LETTER I WITH GRAVE
-<combining_grave> <I>            	: "Ì"   U00CC # LATIN CAPITAL LETTER I WITH GRAVE
-<dead_acute> <I>                 	: "Í"   U00CD # LATIN CAPITAL LETTER I WITH ACUTE
-<Multi_key> <acute> <I>          	: "Í"   U00CD # LATIN CAPITAL LETTER I WITH ACUTE
-<Multi_key> <apostrophe> <I>     	: "Í"   U00CD # LATIN CAPITAL LETTER I WITH ACUTE
-<combining_acute> <I>            	: "Í"   U00CD # LATIN CAPITAL LETTER I WITH ACUTE
-<dead_circumflex> <I>            	: "Î"   U00CE # LATIN CAPITAL LETTER I WITH CIRCUMFLEX
-<Multi_key> <asciicircum> <I>    	: "Î"   U00CE # LATIN CAPITAL LETTER I WITH CIRCUMFLEX
-<dead_diaeresis> <I>             	: "Ï"   U00CF # LATIN CAPITAL LETTER I WITH DIAERESIS
-<Multi_key> <quotedbl> <I>       	: "Ï"   U00CF # LATIN CAPITAL LETTER I WITH DIAERESIS
-<Multi_key> <D> <H>              	: "Ð"   U00D0 # LATIN CAPITAL LETTER ETH
-<dead_tilde> <N>                 	: "Ñ"   U00D1 # LATIN CAPITAL LETTER N WITH TILDE
-<Multi_key> <asciitilde> <N>     	: "Ñ"   U00D1 # LATIN CAPITAL LETTER N WITH TILDE
-<combining_tilde> <N>            	: "Ñ"   U00D1 # LATIN CAPITAL LETTER N WITH TILDE
-<dead_grave> <O>                 	: "Ò"   U00D2 # LATIN CAPITAL LETTER O WITH GRAVE
-<Multi_key> <grave> <O>          	: "Ò"   U00D2 # LATIN CAPITAL LETTER O WITH GRAVE
-<combining_grave> <O>            	: "Ò"   U00D2 # LATIN CAPITAL LETTER O WITH GRAVE
-<dead_acute> <O>                 	: "Ó"   U00D3 # LATIN CAPITAL LETTER O WITH ACUTE
-<Multi_key> <acute> <O>          	: "Ó"   U00D3 # LATIN CAPITAL LETTER O WITH ACUTE
-<Multi_key> <apostrophe> <O>     	: "Ó"   U00D3 # LATIN CAPITAL LETTER O WITH ACUTE
-<combining_acute> <O>            	: "Ó"   U00D3 # LATIN CAPITAL LETTER O WITH ACUTE
-<dead_circumflex> <O>            	: "Ô"   U00D4 # LATIN CAPITAL LETTER O WITH CIRCUMFLEX
-<Multi_key> <asciicircum> <O>    	: "Ô"   U00D4 # LATIN CAPITAL LETTER O WITH CIRCUMFLEX
-<dead_tilde> <O>                 	: "Õ"   U00D5 # LATIN CAPITAL LETTER O WITH TILDE
-<Multi_key> <asciitilde> <O>     	: "Õ"   U00D5 # LATIN CAPITAL LETTER O WITH TILDE
-<combining_tilde> <O>            	: "Õ"   U00D5 # LATIN CAPITAL LETTER O WITH TILDE
-<dead_diaeresis> <O>             	: "Ö"   U00D6 # LATIN CAPITAL LETTER O WITH DIAERESIS
-<Multi_key> <quotedbl> <O>       	: "Ö"   U00D6 # LATIN CAPITAL LETTER O WITH DIAERESIS
-<Multi_key> <x> <x>              	: "×"   U00D7 # MULTIPLICATION SIGN
-<Multi_key> <slash> <O>          	: "Ø"   U00D8 # LATIN CAPITAL LETTER O WITH STROKE
-<Multi_key> <KP_Divide> <O>      	: "Ø"   U00D8 # LATIN CAPITAL LETTER O WITH STROKE
-<dead_grave> <U>                 	: "Ù"   U00D9 # LATIN CAPITAL LETTER U WITH GRAVE
-<Multi_key> <grave> <U>          	: "Ù"   U00D9 # LATIN CAPITAL LETTER U WITH GRAVE
-<combining_grave> <U>            	: "Ù"   U00D9 # LATIN CAPITAL LETTER U WITH GRAVE
-<dead_acute> <U>                 	: "Ú"   U00DA # LATIN CAPITAL LETTER U WITH ACUTE
-<Multi_key> <acute> <U>          	: "Ú"   U00DA # LATIN CAPITAL LETTER U WITH ACUTE
-<Multi_key> <apostrophe> <U>     	: "Ú"   U00DA # LATIN CAPITAL LETTER U WITH ACUTE
-<combining_acute> <U>            	: "Ú"   U00DA # LATIN CAPITAL LETTER U WITH ACUTE
-<dead_circumflex> <U>            	: "Û"   U00DB # LATIN CAPITAL LETTER U WITH CIRCUMFLEX
-<Multi_key> <asciicircum> <U>    	: "Û"   U00DB # LATIN CAPITAL LETTER U WITH CIRCUMFLEX
-<dead_diaeresis> <U>             	: "Ü"   U00DC # LATIN CAPITAL LETTER U WITH DIAERESIS
-<Multi_key> <quotedbl> <U>       	: "Ü"   U00DC # LATIN CAPITAL LETTER U WITH DIAERESIS
-<dead_acute> <Y>                 	: "Ý"   U00DD # LATIN CAPITAL LETTER Y WITH ACUTE
-<Multi_key> <acute> <Y>          	: "Ý"   U00DD # LATIN CAPITAL LETTER Y WITH ACUTE
-<Multi_key> <apostrophe> <Y>     	: "Ý"   U00DD # LATIN CAPITAL LETTER Y WITH ACUTE
-<combining_acute> <Y>            	: "Ý"   U00DD # LATIN CAPITAL LETTER Y WITH ACUTE
-<Multi_key> <T> <H>              	: "Þ"   U00DE # LATIN CAPITAL LETTER THORN
-<dead_grave> <a>                 	: "à"   U00E0 # LATIN SMALL LETTER A WITH GRAVE
-<Multi_key> <grave> <a>          	: "à"   U00E0 # LATIN SMALL LETTER A WITH GRAVE
-<combining_grave> <a>            	: "à"   U00E0 # LATIN SMALL LETTER A WITH GRAVE
-<dead_acute> <a>                 	: "á"   U00E1 # LATIN SMALL LETTER A WITH ACUTE
-<Multi_key> <acute> <a>          	: "á"   U00E1 # LATIN SMALL LETTER A WITH ACUTE
-<Multi_key> <apostrophe> <a>     	: "á"   U00E1 # LATIN SMALL LETTER A WITH ACUTE
-<combining_acute> <a>            	: "á"   U00E1 # LATIN SMALL LETTER A WITH ACUTE
-<dead_circumflex> <a>            	: "â"   U00E2 # LATIN SMALL LETTER A WITH CIRCUMFLEX
-<Multi_key> <asciicircum> <a>    	: "â"   U00E2 # LATIN SMALL LETTER A WITH CIRCUMFLEX
-<dead_tilde> <a>                 	: "ã"   U00E3 # LATIN SMALL LETTER A WITH TILDE
-<Multi_key> <asciitilde> <a>     	: "ã"   U00E3 # LATIN SMALL LETTER A WITH TILDE
-<combining_tilde> <a>            	: "ã"   U00E3 # LATIN SMALL LETTER A WITH TILDE
-<dead_diaeresis> <a>             	: "ä"   U00E4 # LATIN SMALL LETTER A WITH DIAERESIS
-<Multi_key> <quotedbl> <a>       	: "ä"   U00E4 # LATIN SMALL LETTER A WITH DIAERESIS
-<dead_abovering> <a>             	: "å"   U00E5 # LATIN SMALL LETTER A WITH RING ABOVE
-<Multi_key> <o> <a>              	: "å"   U00E5 # LATIN SMALL LETTER A WITH RING ABOVE
-<dead_cedilla> <c>               	: "ç"   U00E7 # LATIN SMALL LETTER C WITH CEDILLA
-<Multi_key> <comma> <c>          	: "ç"   U00E7 # LATIN SMALL LETTER C WITH CEDILLA
-<dead_grave> <e>                 	: "è"   U00E8 # LATIN SMALL LETTER E WITH GRAVE
-<Multi_key> <grave> <e>          	: "è"   U00E8 # LATIN SMALL LETTER E WITH GRAVE
-<combining_grave> <e>            	: "è"   U00E8 # LATIN SMALL LETTER E WITH GRAVE
-<dead_acute> <e>                 	: "é"   U00E9 # LATIN SMALL LETTER E WITH ACUTE
-<Multi_key> <acute> <e>          	: "é"   U00E9 # LATIN SMALL LETTER E WITH ACUTE
-<Multi_key> <apostrophe> <e>     	: "é"   U00E9 # LATIN SMALL LETTER E WITH ACUTE
-<combining_acute> <e>            	: "é"   U00E9 # LATIN SMALL LETTER E WITH ACUTE
-<dead_circumflex> <e>            	: "ê"   U00EA # LATIN SMALL LETTER E WITH CIRCUMFLEX
-<Multi_key> <asciicircum> <e>    	: "ê"   U00EA # LATIN SMALL LETTER E WITH CIRCUMFLEX
-<dead_diaeresis> <e>             	: "ë"   U00EB # LATIN SMALL LETTER E WITH DIAERESIS
-<Multi_key> <quotedbl> <e>       	: "ë"   U00EB # LATIN SMALL LETTER E WITH DIAERESIS
-<dead_grave> <i>                 	: "ì"   U00EC # LATIN SMALL LETTER I WITH GRAVE
-<Multi_key> <grave> <i>          	: "ì"   U00EC # LATIN SMALL LETTER I WITH GRAVE
-<combining_grave> <i>            	: "ì"   U00EC # LATIN SMALL LETTER I WITH GRAVE
-<dead_acute> <i>                 	: "í"   U00ED # LATIN SMALL LETTER I WITH ACUTE
-<Multi_key> <acute> <i>          	: "í"   U00ED # LATIN SMALL LETTER I WITH ACUTE
-<Multi_key> <apostrophe> <i>     	: "í"   U00ED # LATIN SMALL LETTER I WITH ACUTE
-<combining_acute> <i>            	: "í"   U00ED # LATIN SMALL LETTER I WITH ACUTE
-<dead_circumflex> <i>            	: "î"   U00EE # LATIN SMALL LETTER I WITH CIRCUMFLEX
-<Multi_key> <asciicircum> <i>    	: "î"   U00EE # LATIN SMALL LETTER I WITH CIRCUMFLEX
-<dead_diaeresis> <i>             	: "ï"   U00EF # LATIN SMALL LETTER I WITH DIAERESIS
-<Multi_key> <quotedbl> <i>       	: "ï"   U00EF # LATIN SMALL LETTER I WITH DIAERESIS
-<Multi_key> <d> <h>              	: "ð"   U00F0 # LATIN SMALL LETTER ETH
-<dead_tilde> <n>                 	: "ñ"   U00F1 # LATIN SMALL LETTER N WITH TILDE
-<Multi_key> <asciitilde> <n>     	: "ñ"   U00F1 # LATIN SMALL LETTER N WITH TILDE
-<combining_tilde> <n>            	: "ñ"   U00F1 # LATIN SMALL LETTER N WITH TILDE
-<dead_grave> <o>                 	: "ò"   U00F2 # LATIN SMALL LETTER O WITH GRAVE
-<Multi_key> <grave> <o>          	: "ò"   U00F2 # LATIN SMALL LETTER O WITH GRAVE
-<combining_grave> <o>            	: "ò"   U00F2 # LATIN SMALL LETTER O WITH GRAVE
-<dead_acute> <o>                 	: "ó"   U00F3 # LATIN SMALL LETTER O WITH ACUTE
-<Multi_key> <acute> <o>          	: "ó"   U00F3 # LATIN SMALL LETTER O WITH ACUTE
-<Multi_key> <apostrophe> <o>     	: "ó"   U00F3 # LATIN SMALL LETTER O WITH ACUTE
-<combining_acute> <o>            	: "ó"   U00F3 # LATIN SMALL LETTER O WITH ACUTE
-<dead_circumflex> <o>            	: "ô"   U00F4 # LATIN SMALL LETTER O WITH CIRCUMFLEX
-<Multi_key> <asciicircum> <o>    	: "ô"   U00F4 # LATIN SMALL LETTER O WITH CIRCUMFLEX
-<dead_tilde> <o>                 	: "õ"   U00F5 # LATIN SMALL LETTER O WITH TILDE
-<Multi_key> <asciitilde> <o>     	: "õ"   U00F5 # LATIN SMALL LETTER O WITH TILDE
-<combining_tilde> <o>            	: "õ"   U00F5 # LATIN SMALL LETTER O WITH TILDE
-<dead_diaeresis> <o>             	: "ö"   U00F6 # LATIN SMALL LETTER O WITH DIAERESIS
-<Multi_key> <quotedbl> <o>       	: "ö"   U00F6 # LATIN SMALL LETTER O WITH DIAERESIS
-<Multi_key> <colon> <minus>      	: "÷"   U00F7 # DIVISION SIGN
-<Multi_key> <minus> <colon>      	: "÷"   U00F7 # DIVISION SIGN
-<Multi_key> <slash> <o>          	: "ø"   U00F8 # LATIN SMALL LETTER O WITH STROKE
-<Multi_key> <KP_Divide> <o>      	: "ø"   U00F8 # LATIN SMALL LETTER O WITH STROKE
-<dead_grave> <u>                 	: "ù"   U00F9 # LATIN SMALL LETTER U WITH GRAVE
-<Multi_key> <grave> <u>          	: "ù"   U00F9 # LATIN SMALL LETTER U WITH GRAVE
-<combining_grave> <u>            	: "ù"   U00F9 # LATIN SMALL LETTER U WITH GRAVE
-<dead_acute> <u>                 	: "ú"   U00FA # LATIN SMALL LETTER U WITH ACUTE
-<Multi_key> <acute> <u>          	: "ú"   U00FA # LATIN SMALL LETTER U WITH ACUTE
-<Multi_key> <apostrophe> <u>     	: "ú"   U00FA # LATIN SMALL LETTER U WITH ACUTE
-<combining_acute> <u>            	: "ú"   U00FA # LATIN SMALL LETTER U WITH ACUTE
-<dead_circumflex> <u>            	: "û"   U00FB # LATIN SMALL LETTER U WITH CIRCUMFLEX
-<Multi_key> <asciicircum> <u>    	: "û"   U00FB # LATIN SMALL LETTER U WITH CIRCUMFLEX
-<dead_diaeresis> <u>             	: "ü"   U00FC # LATIN SMALL LETTER U WITH DIAERESIS
-<Multi_key> <quotedbl> <u>       	: "ü"   U00FC # LATIN SMALL LETTER U WITH DIAERESIS
-<dead_acute> <y>                 	: "ý"   U00FD # LATIN SMALL LETTER Y WITH ACUTE
-<Multi_key> <acute> <y>          	: "ý"   U00FD # LATIN SMALL LETTER Y WITH ACUTE
-<Multi_key> <apostrophe> <y>     	: "ý"   U00FD # LATIN SMALL LETTER Y WITH ACUTE
-<combining_acute> <y>            	: "ý"   U00FD # LATIN SMALL LETTER Y WITH ACUTE
-<Multi_key> <t> <h>              	: "þ"   U00FE # LATIN SMALL LETTER THORN
-<dead_diaeresis> <y>             	: "ÿ"   U00FF # LATIN SMALL LETTER Y WITH DIAERESIS
-<Multi_key> <quotedbl> <y>       	: "ÿ"   U00FF # LATIN SMALL LETTER Y WITH DIAERESIS
+<Multi_key> <comma> <minus>      	: "¬"   notsign # NOT SIGN
+<Multi_key> <minus> <comma>      	: "¬"   notsign # NOT SIGN
+<dead_circumflex> <Multi_key> <underscore> <a> 	: "ª"   ordfeminine # FEMININE ORDINAL INDICATOR
+<Multi_key> <asciicircum> <underscore> <a> 	: "ª"   ordfeminine # FEMININE ORDINAL INDICATOR
+<dead_circumflex> <Multi_key> <underbar> <a> 	: "ª"   ordfeminine # FEMININE ORDINAL INDICATOR
+<Multi_key> <asciicircum> <underbar> <a> 	: "ª"   ordfeminine # FEMININE ORDINAL INDICATOR
+<dead_circumflex> <2>            	: "²"   twosuperior # SUPERSCRIPT TWO
+<Multi_key> <asciicircum> <2>    	: "²"   twosuperior # SUPERSCRIPT TWO
+<dead_circumflex> <KP_Space>     	: "²"   twosuperior # SUPERSCRIPT TWO
+<Multi_key> <asciicircum> <KP_Space> 	: "²"   twosuperior # SUPERSCRIPT TWO
+<dead_circumflex> <KP_2>         	: "²"   twosuperior # SUPERSCRIPT TWO
+<Multi_key> <asciicircum> <KP_2> 	: "²"   twosuperior # SUPERSCRIPT TWO
+<dead_circumflex> <3>            	: "³"   threesuperior # SUPERSCRIPT THREE
+<Multi_key> <asciicircum> <3>    	: "³"   threesuperior # SUPERSCRIPT THREE
+<dead_circumflex> <KP_3>         	: "³"   threesuperior # SUPERSCRIPT THREE
+<Multi_key> <asciicircum> <KP_3> 	: "³"   threesuperior # SUPERSCRIPT THREE
+<Multi_key> <m> <u>              	: "µ"   mu # MICRO SIGN
+<dead_circumflex> <1>            	: "¹"   onesuperior # SUPERSCRIPT ONE
+<Multi_key> <asciicircum> <1>    	: "¹"   onesuperior # SUPERSCRIPT ONE
+<dead_circumflex> <KP_1>         	: "¹"   onesuperior # SUPERSCRIPT ONE
+<Multi_key> <asciicircum> <KP_1> 	: "¹"   onesuperior # SUPERSCRIPT ONE
+<dead_circumflex> <Multi_key> <underscore> <o> 	: "º"   masculine # MASCULINE ORDINAL INDICATOR
+<Multi_key> <asciicircum> <underscore> <o> 	: "º"   masculine # MASCULINE ORDINAL INDICATOR
+<dead_circumflex> <Multi_key> <underbar> <o> 	: "º"   masculine # MASCULINE ORDINAL INDICATOR
+<Multi_key> <asciicircum> <underbar> <o> 	: "º"   masculine # MASCULINE ORDINAL INDICATOR
+<Multi_key> <1> <4>              	: "¼"   onequarter # VULGAR FRACTION ONE QUARTER
+<Multi_key> <1> <2>              	: "½"   onehalf # VULGAR FRACTION ONE HALF
+<Multi_key> <3> <4>              	: "¾"   threequarters # VULGAR FRACTION THREE QUARTERS
+<dead_grave> <A>                 	: "À"   Agrave # LATIN CAPITAL LETTER A WITH GRAVE
+<Multi_key> <grave> <A>          	: "À"   Agrave # LATIN CAPITAL LETTER A WITH GRAVE
+<combining_grave> <A>            	: "À"   Agrave # LATIN CAPITAL LETTER A WITH GRAVE
+<dead_acute> <A>                 	: "Á"   Aacute # LATIN CAPITAL LETTER A WITH ACUTE
+<Multi_key> <acute> <A>          	: "Á"   Aacute # LATIN CAPITAL LETTER A WITH ACUTE
+<Multi_key> <apostrophe> <A>     	: "Á"   Aacute # LATIN CAPITAL LETTER A WITH ACUTE
+<combining_acute> <A>            	: "Á"   Aacute # LATIN CAPITAL LETTER A WITH ACUTE
+<dead_circumflex> <A>            	: "Â"   Acircumflex # LATIN CAPITAL LETTER A WITH CIRCUMFLEX
+<Multi_key> <asciicircum> <A>    	: "Â"   Acircumflex # LATIN CAPITAL LETTER A WITH CIRCUMFLEX
+<dead_tilde> <A>                 	: "Ã"   Atilde # LATIN CAPITAL LETTER A WITH TILDE
+<Multi_key> <asciitilde> <A>     	: "Ã"   Atilde # LATIN CAPITAL LETTER A WITH TILDE
+<combining_tilde> <A>            	: "Ã"   Atilde # LATIN CAPITAL LETTER A WITH TILDE
+<dead_diaeresis> <A>             	: "Ä"   Adiaeresis # LATIN CAPITAL LETTER A WITH DIAERESIS
+<Multi_key> <quotedbl> <A>       	: "Ä"   Adiaeresis # LATIN CAPITAL LETTER A WITH DIAERESIS
+<dead_abovering> <A>             	: "Å"   Aring # LATIN CAPITAL LETTER A WITH RING ABOVE
+<Multi_key> <o> <A>              	: "Å"   Aring # LATIN CAPITAL LETTER A WITH RING ABOVE
+<dead_cedilla> <C>               	: "Ç"   Ccedilla # LATIN CAPITAL LETTER C WITH CEDILLA
+<Multi_key> <comma> <C>          	: "Ç"   Ccedilla # LATIN CAPITAL LETTER C WITH CEDILLA
+<dead_grave> <E>                 	: "È"   Egrave # LATIN CAPITAL LETTER E WITH GRAVE
+<Multi_key> <grave> <E>          	: "È"   Egrave # LATIN CAPITAL LETTER E WITH GRAVE
+<combining_grave> <E>            	: "È"   Egrave # LATIN CAPITAL LETTER E WITH GRAVE
+<dead_acute> <E>                 	: "É"   Eacute # LATIN CAPITAL LETTER E WITH ACUTE
+<Multi_key> <acute> <E>          	: "É"   Eacute # LATIN CAPITAL LETTER E WITH ACUTE
+<Multi_key> <apostrophe> <E>     	: "É"   Eacute # LATIN CAPITAL LETTER E WITH ACUTE
+<combining_acute> <E>            	: "É"   Eacute # LATIN CAPITAL LETTER E WITH ACUTE
+<dead_circumflex> <E>            	: "Ê"   Ecircumflex # LATIN CAPITAL LETTER E WITH CIRCUMFLEX
+<Multi_key> <asciicircum> <E>    	: "Ê"   Ecircumflex # LATIN CAPITAL LETTER E WITH CIRCUMFLEX
+<dead_diaeresis> <E>             	: "Ë"   Ediaeresis # LATIN CAPITAL LETTER E WITH DIAERESIS
+<Multi_key> <quotedbl> <E>       	: "Ë"   Ediaeresis # LATIN CAPITAL LETTER E WITH DIAERESIS
+<dead_grave> <I>                 	: "Ì"   Igrave # LATIN CAPITAL LETTER I WITH GRAVE
+<Multi_key> <grave> <I>          	: "Ì"   Igrave # LATIN CAPITAL LETTER I WITH GRAVE
+<combining_grave> <I>            	: "Ì"   Igrave # LATIN CAPITAL LETTER I WITH GRAVE
+<dead_acute> <I>                 	: "Í"   Iacute # LATIN CAPITAL LETTER I WITH ACUTE
+<Multi_key> <acute> <I>          	: "Í"   Iacute # LATIN CAPITAL LETTER I WITH ACUTE
+<Multi_key> <apostrophe> <I>     	: "Í"   Iacute # LATIN CAPITAL LETTER I WITH ACUTE
+<combining_acute> <I>            	: "Í"   Iacute # LATIN CAPITAL LETTER I WITH ACUTE
+<dead_circumflex> <I>            	: "Î"   Icircumflex # LATIN CAPITAL LETTER I WITH CIRCUMFLEX
+<Multi_key> <asciicircum> <I>    	: "Î"   Icircumflex # LATIN CAPITAL LETTER I WITH CIRCUMFLEX
+<dead_diaeresis> <I>             	: "Ï"   Idiaeresis # LATIN CAPITAL LETTER I WITH DIAERESIS
+<Multi_key> <quotedbl> <I>       	: "Ï"   Idiaeresis # LATIN CAPITAL LETTER I WITH DIAERESIS
+<Multi_key> <D> <H>              	: "Ð"   ETH # LATIN CAPITAL LETTER ETH
+<dead_tilde> <N>                 	: "Ñ"   Ntilde # LATIN CAPITAL LETTER N WITH TILDE
+<Multi_key> <asciitilde> <N>     	: "Ñ"   Ntilde # LATIN CAPITAL LETTER N WITH TILDE
+<combining_tilde> <N>            	: "Ñ"   Ntilde # LATIN CAPITAL LETTER N WITH TILDE
+<dead_grave> <O>                 	: "Ò"   Ograve # LATIN CAPITAL LETTER O WITH GRAVE
+<Multi_key> <grave> <O>          	: "Ò"   Ograve # LATIN CAPITAL LETTER O WITH GRAVE
+<combining_grave> <O>            	: "Ò"   Ograve # LATIN CAPITAL LETTER O WITH GRAVE
+<dead_acute> <O>                 	: "Ó"   Oacute # LATIN CAPITAL LETTER O WITH ACUTE
+<Multi_key> <acute> <O>          	: "Ó"   Oacute # LATIN CAPITAL LETTER O WITH ACUTE
+<Multi_key> <apostrophe> <O>     	: "Ó"   Oacute # LATIN CAPITAL LETTER O WITH ACUTE
+<combining_acute> <O>            	: "Ó"   Oacute # LATIN CAPITAL LETTER O WITH ACUTE
+<dead_circumflex> <O>            	: "Ô"   Ocircumflex # LATIN CAPITAL LETTER O WITH CIRCUMFLEX
+<Multi_key> <asciicircum> <O>    	: "Ô"   Ocircumflex # LATIN CAPITAL LETTER O WITH CIRCUMFLEX
+<dead_tilde> <O>                 	: "Õ"   Otilde # LATIN CAPITAL LETTER O WITH TILDE
+<Multi_key> <asciitilde> <O>     	: "Õ"   Otilde # LATIN CAPITAL LETTER O WITH TILDE
+<combining_tilde> <O>            	: "Õ"   Otilde # LATIN CAPITAL LETTER O WITH TILDE
+<dead_diaeresis> <O>             	: "Ö"   Odiaeresis # LATIN CAPITAL LETTER O WITH DIAERESIS
+<Multi_key> <quotedbl> <O>       	: "Ö"   Odiaeresis # LATIN CAPITAL LETTER O WITH DIAERESIS
+<Multi_key> <x> <x>              	: "×"   multiply # MULTIPLICATION SIGN
+<Multi_key> <slash> <O>          	: "Ø"   Oslash # LATIN CAPITAL LETTER O WITH STROKE
+<Multi_key> <KP_Divide> <O>      	: "Ø"   Oslash # LATIN CAPITAL LETTER O WITH STROKE
+<dead_grave> <U>                 	: "Ù"   Ugrave # LATIN CAPITAL LETTER U WITH GRAVE
+<Multi_key> <grave> <U>          	: "Ù"   Ugrave # LATIN CAPITAL LETTER U WITH GRAVE
+<combining_grave> <U>            	: "Ù"   Ugrave # LATIN CAPITAL LETTER U WITH GRAVE
+<dead_acute> <U>                 	: "Ú"   Uacute # LATIN CAPITAL LETTER U WITH ACUTE
+<Multi_key> <acute> <U>          	: "Ú"   Uacute # LATIN CAPITAL LETTER U WITH ACUTE
+<Multi_key> <apostrophe> <U>     	: "Ú"   Uacute # LATIN CAPITAL LETTER U WITH ACUTE
+<combining_acute> <U>            	: "Ú"   Uacute # LATIN CAPITAL LETTER U WITH ACUTE
+<dead_circumflex> <U>            	: "Û"   Ucircumflex # LATIN CAPITAL LETTER U WITH CIRCUMFLEX
+<Multi_key> <asciicircum> <U>    	: "Û"   Ucircumflex # LATIN CAPITAL LETTER U WITH CIRCUMFLEX
+<dead_diaeresis> <U>             	: "Ü"   Udiaeresis # LATIN CAPITAL LETTER U WITH DIAERESIS
+<Multi_key> <quotedbl> <U>       	: "Ü"   Udiaeresis # LATIN CAPITAL LETTER U WITH DIAERESIS
+<dead_acute> <Y>                 	: "Ý"   Yacute # LATIN CAPITAL LETTER Y WITH ACUTE
+<Multi_key> <acute> <Y>          	: "Ý"   Yacute # LATIN CAPITAL LETTER Y WITH ACUTE
+<Multi_key> <apostrophe> <Y>     	: "Ý"   Yacute # LATIN CAPITAL LETTER Y WITH ACUTE
+<combining_acute> <Y>            	: "Ý"   Yacute # LATIN CAPITAL LETTER Y WITH ACUTE
+<Multi_key> <T> <H>              	: "Þ"   THORN # LATIN CAPITAL LETTER THORN
+<dead_grave> <a>                 	: "à"   agrave # LATIN SMALL LETTER A WITH GRAVE
+<Multi_key> <grave> <a>          	: "à"   agrave # LATIN SMALL LETTER A WITH GRAVE
+<combining_grave> <a>            	: "à"   agrave # LATIN SMALL LETTER A WITH GRAVE
+<dead_acute> <a>                 	: "á"   aacute # LATIN SMALL LETTER A WITH ACUTE
+<Multi_key> <acute> <a>          	: "á"   aacute # LATIN SMALL LETTER A WITH ACUTE
+<Multi_key> <apostrophe> <a>     	: "á"   aacute # LATIN SMALL LETTER A WITH ACUTE
+<combining_acute> <a>            	: "á"   aacute # LATIN SMALL LETTER A WITH ACUTE
+<dead_circumflex> <a>            	: "â"   acircumflex # LATIN SMALL LETTER A WITH CIRCUMFLEX
+<Multi_key> <asciicircum> <a>    	: "â"   acircumflex # LATIN SMALL LETTER A WITH CIRCUMFLEX
+<dead_tilde> <a>                 	: "ã"   atilde # LATIN SMALL LETTER A WITH TILDE
+<Multi_key> <asciitilde> <a>     	: "ã"   atilde # LATIN SMALL LETTER A WITH TILDE
+<combining_tilde> <a>            	: "ã"   atilde # LATIN SMALL LETTER A WITH TILDE
+<dead_diaeresis> <a>             	: "ä"   adiaeresis # LATIN SMALL LETTER A WITH DIAERESIS
+<Multi_key> <quotedbl> <a>       	: "ä"   adiaeresis # LATIN SMALL LETTER A WITH DIAERESIS
+<dead_abovering> <a>             	: "å"   aring # LATIN SMALL LETTER A WITH RING ABOVE
+<Multi_key> <o> <a>              	: "å"   aring # LATIN SMALL LETTER A WITH RING ABOVE
+<dead_cedilla> <c>               	: "ç"   ccedilla # LATIN SMALL LETTER C WITH CEDILLA
+<Multi_key> <comma> <c>          	: "ç"   ccedilla # LATIN SMALL LETTER C WITH CEDILLA
+<dead_grave> <e>                 	: "è"   egrave # LATIN SMALL LETTER E WITH GRAVE
+<Multi_key> <grave> <e>          	: "è"   egrave # LATIN SMALL LETTER E WITH GRAVE
+<combining_grave> <e>            	: "è"   egrave # LATIN SMALL LETTER E WITH GRAVE
+<dead_acute> <e>                 	: "é"   eacute # LATIN SMALL LETTER E WITH ACUTE
+<Multi_key> <acute> <e>          	: "é"   eacute # LATIN SMALL LETTER E WITH ACUTE
+<Multi_key> <apostrophe> <e>     	: "é"   eacute # LATIN SMALL LETTER E WITH ACUTE
+<combining_acute> <e>            	: "é"   eacute # LATIN SMALL LETTER E WITH ACUTE
+<dead_circumflex> <e>            	: "ê"   ecircumflex # LATIN SMALL LETTER E WITH CIRCUMFLEX
+<Multi_key> <asciicircum> <e>    	: "ê"   ecircumflex # LATIN SMALL LETTER E WITH CIRCUMFLEX
+<dead_diaeresis> <e>             	: "ë"   ediaeresis # LATIN SMALL LETTER E WITH DIAERESIS
+<Multi_key> <quotedbl> <e>       	: "ë"   ediaeresis # LATIN SMALL LETTER E WITH DIAERESIS
+<dead_grave> <i>                 	: "ì"   igrave # LATIN SMALL LETTER I WITH GRAVE
+<Multi_key> <grave> <i>          	: "ì"   igrave # LATIN SMALL LETTER I WITH GRAVE
+<combining_grave> <i>            	: "ì"   igrave # LATIN SMALL LETTER I WITH GRAVE
+<dead_acute> <i>                 	: "í"   iacute # LATIN SMALL LETTER I WITH ACUTE
+<Multi_key> <acute> <i>          	: "í"   iacute # LATIN SMALL LETTER I WITH ACUTE
+<Multi_key> <apostrophe> <i>     	: "í"   iacute # LATIN SMALL LETTER I WITH ACUTE
+<combining_acute> <i>            	: "í"   iacute # LATIN SMALL LETTER I WITH ACUTE
+<dead_circumflex> <i>            	: "î"   icircumflex # LATIN SMALL LETTER I WITH CIRCUMFLEX
+<Multi_key> <asciicircum> <i>    	: "î"   icircumflex # LATIN SMALL LETTER I WITH CIRCUMFLEX
+<dead_diaeresis> <i>             	: "ï"   idiaeresis # LATIN SMALL LETTER I WITH DIAERESIS
+<Multi_key> <quotedbl> <i>       	: "ï"   idiaeresis # LATIN SMALL LETTER I WITH DIAERESIS
+<Multi_key> <d> <h>              	: "ð"   eth # LATIN SMALL LETTER ETH
+<dead_tilde> <n>                 	: "ñ"   ntilde # LATIN SMALL LETTER N WITH TILDE
+<Multi_key> <asciitilde> <n>     	: "ñ"   ntilde # LATIN SMALL LETTER N WITH TILDE
+<combining_tilde> <n>            	: "ñ"   ntilde # LATIN SMALL LETTER N WITH TILDE
+<dead_grave> <o>                 	: "ò"   ograve # LATIN SMALL LETTER O WITH GRAVE
+<Multi_key> <grave> <o>          	: "ò"   ograve # LATIN SMALL LETTER O WITH GRAVE
+<combining_grave> <o>            	: "ò"   ograve # LATIN SMALL LETTER O WITH GRAVE
+<dead_acute> <o>                 	: "ó"   oacute # LATIN SMALL LETTER O WITH ACUTE
+<Multi_key> <acute> <o>          	: "ó"   oacute # LATIN SMALL LETTER O WITH ACUTE
+<Multi_key> <apostrophe> <o>     	: "ó"   oacute # LATIN SMALL LETTER O WITH ACUTE
+<combining_acute> <o>            	: "ó"   oacute # LATIN SMALL LETTER O WITH ACUTE
+<dead_circumflex> <o>            	: "ô"   ocircumflex # LATIN SMALL LETTER O WITH CIRCUMFLEX
+<Multi_key> <asciicircum> <o>    	: "ô"   ocircumflex # LATIN SMALL LETTER O WITH CIRCUMFLEX
+<dead_tilde> <o>                 	: "õ"   otilde # LATIN SMALL LETTER O WITH TILDE
+<Multi_key> <asciitilde> <o>     	: "õ"   otilde # LATIN SMALL LETTER O WITH TILDE
+<combining_tilde> <o>            	: "õ"   otilde # LATIN SMALL LETTER O WITH TILDE
+<dead_diaeresis> <o>             	: "ö"   odiaeresis # LATIN SMALL LETTER O WITH DIAERESIS
+<Multi_key> <quotedbl> <o>       	: "ö"   odiaeresis # LATIN SMALL LETTER O WITH DIAERESIS
+<Multi_key> <colon> <minus>      	: "÷"   division # DIVISION SIGN
+<Multi_key> <minus> <colon>      	: "÷"   division # DIVISION SIGN
+<Multi_key> <slash> <o>          	: "ø"   oslash # LATIN SMALL LETTER O WITH STROKE
+<Multi_key> <KP_Divide> <o>      	: "ø"   oslash # LATIN SMALL LETTER O WITH STROKE
+<dead_grave> <u>                 	: "ù"   ugrave # LATIN SMALL LETTER U WITH GRAVE
+<Multi_key> <grave> <u>          	: "ù"   ugrave # LATIN SMALL LETTER U WITH GRAVE
+<combining_grave> <u>            	: "ù"   ugrave # LATIN SMALL LETTER U WITH GRAVE
+<dead_acute> <u>                 	: "ú"   uacute # LATIN SMALL LETTER U WITH ACUTE
+<Multi_key> <acute> <u>          	: "ú"   uacute # LATIN SMALL LETTER U WITH ACUTE
+<Multi_key> <apostrophe> <u>     	: "ú"   uacute # LATIN SMALL LETTER U WITH ACUTE
+<combining_acute> <u>            	: "ú"   uacute # LATIN SMALL LETTER U WITH ACUTE
+<dead_circumflex> <u>            	: "û"   ucircumflex # LATIN SMALL LETTER U WITH CIRCUMFLEX
+<Multi_key> <asciicircum> <u>    	: "û"   ucircumflex # LATIN SMALL LETTER U WITH CIRCUMFLEX
+<dead_diaeresis> <u>             	: "ü"   udiaeresis # LATIN SMALL LETTER U WITH DIAERESIS
+<Multi_key> <quotedbl> <u>       	: "ü"   udiaeresis # LATIN SMALL LETTER U WITH DIAERESIS
+<dead_acute> <y>                 	: "ý"   yacute # LATIN SMALL LETTER Y WITH ACUTE
+<Multi_key> <acute> <y>          	: "ý"   yacute # LATIN SMALL LETTER Y WITH ACUTE
+<Multi_key> <apostrophe> <y>     	: "ý"   yacute # LATIN SMALL LETTER Y WITH ACUTE
+<combining_acute> <y>            	: "ý"   yacute # LATIN SMALL LETTER Y WITH ACUTE
+<Multi_key> <t> <h>              	: "þ"   thorn # LATIN SMALL LETTER THORN
+<dead_diaeresis> <y>             	: "ÿ"   ydiaeresis # LATIN SMALL LETTER Y WITH DIAERESIS
+<Multi_key> <quotedbl> <y>       	: "ÿ"   ydiaeresis # LATIN SMALL LETTER Y WITH DIAERESIS
 <dead_macron> <A>                	: "Ā"   U0100 # LATIN CAPITAL LETTER A WITH MACRON
 <Multi_key> <macron> <A>         	: "Ā"   U0100 # LATIN CAPITAL LETTER A WITH MACRON
 <Multi_key> <underscore> <A>     	: "Ā"   U0100 # LATIN CAPITAL LETTER A WITH MACRON
